src/containers/singleEvent/index.tsx

```diff
@@ -77,6 +120,14 @@ const SingleEvent: React.FC<SingleEventProps> = ({ events }) => {
         />
       </Helmet>
       <ContentContainer>
+        {privilegeLevel === PrivilegeLevel.ADMIN ? (
```

Comment:
This should actually be inserted inside the "conditionally render" function on line 98, right now these buttons will be rendered even if the event is loading or has failed.

IMO we should move the conditionally render stuff directly into the JSX, but we can leave that alone for now.

Reply:
Agree Ryan, we should just make that change now and move the subrendering to the main return.